docs/dictionary/property/recordFormat.lcdoc

```diff
@@ -22,29 +22,25 @@ set the recordFormat to the storedSoundFormat of me
 
 The result:
 >*Important:*  If the <recordFormat> is set to "movie", the resulting
-> <file> is in the <QuickTime> file format. <QuickTime> files cannot be
+> file is in the <QuickTime> file format. <QuickTime> files cannot be
 > played as <audio clip|audio clips>. To play such a sound, either
 > create a <player> and set its <filename> <property> to the <file
 > path|path> of the <file> you recorded, or use the play videoclip form
 > of the <play> <command>.
 
-Value (enum):
-
--   aiff: records a file in AIFF format (default)
--   wave: records a file in WAV format
--   ulaw: records a file in AU format
--   movie: records a file in QuickTime format
-
+Value(string): The file format of the created recording.
 
 Description:
 Use the <recordFormat> <property> to specify what kind of file to create
 when recording a sound, depending on how the file will be used.
 
 The <recordFormat> <property> determines what kind of file is created
-when you use the <record sound> <command> to record sound.
+when you use the <record sound> <command> to record sound. Use the 
+<recordFormats> <function> returns a list of supported <recordFormat> 
+values.
 
 Not all sound file formats are compatible with all supported codecs. If
-you try to record sound using a recordCompressionType that is not
+you try to record sound using a <recordCompression> that is not
```

Comment:
`recordCompression` isn't listed in the references. Is this supposed to be a reference to `recordCompressionTypes`?